### PR TITLE
Update readme.md and pre-commit-config.yaml

### LINKED ISSUE #23: Update README.md
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* ***If you have submitted a pull request addressing the same issue, please leave a comment!***

***

### Description

In order to increase collaboration and contribution on the development of this module, it is a good idea to start with enhancing the README file. We do have a template repository which contains several files that will become a standard: https://github.com/traveloka/terraform-aws-modules-template

On that repository you will find a [README](https://github.com/traveloka/terraform-aws-modules-template/blob/master/README.md) file that can be used as an example. What points should be written on the README.

The example [README](https://github.com/traveloka/terraform-aws-modules-template/blob/master/README.md) was created by @Rukaan and reviewed by @rafikurnia, @sumodirjo, and also other @traveloka/site-infra members. See traveloka/terraform-aws-modules-template#9 for more information.

### References

* https://opensource.guide/starting-a-project/

### PR DESCRIPTION
**Pull request by aashari**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

***

Fixes #23 

***

Release note for [CHANGELOG](https://github.com/traveloka/terraform-aws-autoscaling/blob/master/CHANGELOG.md):

```release-note
NOTES:
* Add .pre-commit-config.yaml to include terraform_fmt and terraform_docs
* Update README.md to be informative
```

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.20.0
+  rev: v1.22.0
```
Comment (sumodirjo) [inline .pre-commit-config.yaml:2]: Can we use version `v1.32.0` ?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
   hooks:
     - id: terraform_fmt
-    - id: terraform_tflint
     - id: terraform_docs
-  
\ No newline at end of file
+- repo: git://github.com/pre-commit/pre-commit-hooks
+  rev: v2.4.0
```
Comment (sumodirjo) [inline .pre-commit-config.yaml:7]: also this to `v3.2.0`

**Comment by aashari** [comment]
Mas @sumodirjo, please re review my PR, I have updated the version, thanks

**Review by sumodirjo — APPROVED** [review]

**Review by salvianreynaldi — APPROVED** [review]
LGTM

**Pull request merged**